tests/behat/features/deputy/02-ndr/10-submit.feature

```diff
@@ -94,14 +94,14 @@ Feature: ndr / report submit
     @ndr
     Scenario: check NDR report not accessible after submission
         Given I am logged in as "behat-user-ndr@publicguardian.gov.uk" with password "Abcd1234"
-        And the URL "/ndr/1/visits-care/summary" should not be accessible
-        And the URL "/ndr/1/deputy-expenses/summary" should not be accessible
-        And the URL "/ndr/1/income-benefits/summary" should not be accessible
-        And the URL "/ndr/1/bank-accounts/summary" should not be accessible
-        And the URL "/ndr/1/assets/summary" should not be accessible
-        And the URL "/ndr/1/debts/summary" should not be accessible
-        And the URL "/ndr/1/actions/summary" should not be accessible
-        And the URL "/ndr/1/any-other-info/summary" should not be accessible
+        And the URL "/ndr/5/visits-care/summary" should not be accessible
+        And the URL "/ndr/5/deputy-expenses/summary" should not be accessible
+        And the URL "/ndr/5/income-benefits/summary" should not be accessible
+        And the URL "/ndr/5/bank-accounts/summary" should not be accessible
+        And the URL "/ndr/5/assets/summary" should not be accessible
+        And the URL "/ndr/5/debts/summary" should not be accessible
+        And the URL "/ndr/5/actions/summary" should not be accessible
+        And the URL "/ndr/5/any-other-info/summary" should not be accessible
```

Comment:
Does this still work if running the ndr suite in isolation?

Reply:
Yes, as long as the fixtures are generated before the suite is run (which they currently are). The fixtures now create reports 1–4.